### PR TITLE
Update to new WorkflowClient API

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: activities-examples/src/execute-workflow.ts [hunk 1]
```diff
@@ -5,9 +5,7 @@ async function run(): Promise<void> {
   const connection = new Connection();
   const client = new WorkflowClient(connection.service);
 
-  const handle = client.createWorkflowHandle(httpWorkflow, { taskQueue: 'tutorial' });
-
-  const result = await handle.execute();
+  const result = await client.execute(httpWorkflow, { taskQueue: 'tutorial' });
```
Comment (sw-yx) [inline activities-examples/src/execute-workflow.ts:8]: i thought we were going to recommend people to set defaultTaskQueue?

**Review by sw-yx — APPROVED** [review]
left 1 comment

**Comment by sw-yx** [comment]
tried to swap out the package.json versions with grep and sed but didnt work 😂  `find . -name "package.json" -not -path "*/node_modules/*" |  xargs grep -lr --exclude-dir={node_modules,.next} -e "temporalio" | xargs -0 sed -i '' -e 's/"temporalio": "0.11.x"/"temporalio": "0.13.x"/g'`

**Comment by lorensr** [comment]
Does this count as manual? 😄
image

**Comment by sw-yx** [comment]
yes haha

basically was hoping that this is the day i learn grep/sed but no dice

**Comment by tolzhabayev** [comment]
https://twitter.com/learn_byexample/status/1455043504426520583 works for me

**Comment by sw-yx** [comment]
adding search attributes example
image

**Comment by sw-yx** [comment]
and continue as new exxample 
image

**Comment by sw-yx** [comment]
adding versioning example 
image

**Pull request merged**